### PR TITLE
#13 Document installation instructions

### PR DESCRIPTION
**Pull request by RyanSept**
Address #13

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .HOLMK
 .hollogs
 .holobjs
+HOL
```
Comment (xrchz) [inline .gitignore:4]: is this intentional? I think it's probably better not to recommend to build HOL _within_ the verifereum working directory, but rather externally (e.g. alongside it)

**Review by xrchz — APPROVED** [review]
Looking good for the most part - thanks!
Left a small thing to address

File: README.md [hunk 2]
```diff
@@ -7,12 +8,79 @@ When you clone this repository don't forget to `git submodule update --init` too
...
 Work in progress.
 
+## Getting Started
+
+1.  Install and polyml from source
+
+    i. Download the source code from the polyml github repository
+
+    ```bash
+    wget wget https://github.com/polyml/polyml/archive/refs/heads/master.zip -O poly-master.zip;
```
Comment (xrchz) [inline README.md:18]: are the two `wget`s here correct?

File: .gitignore (hunk 1 above)
Comment (RyanSept) [reply .gitignore]: Gotcha. Wasn't sure how people with multiple per-project HOLs tend to do it.

File: README.md (hunk 2 above)
Comment (RyanSept) [reply README.md]: whoops good catch

**Review by xrchz — APPROVED** [review]

**Pull request merged**